src/values.rs

```diff
@@ -109,9 +132,35 @@ impl Ord for OnOff {
     }
 }
 
-/// An open/closed state. Internal representation may be either
-/// Open or Closed.
-#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+serde_impl!(OnOff(String) {
```

Comment:
__question:__ wondering if it would be beneficial to leave a note here why we do this and maybe add a reference to a serde issue where change is requested? Default behavior is actually really surprising :)